com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -562,12 +562,6 @@ private void InitializeOverlay(IEOSCoroutineOwner coroutineOwner)
             // NOTE: on some platforms the EOS platform is init'd by a native dynamic library. In
             // those cases, this code will early out.
             public void Init(IEOSCoroutineOwner coroutineOwner)
-            {
-                Init(coroutineOwner, EOSPackageInfo.ConfigFileName);
-            }
-
-            //-------------------------------------------------------------------------
-            public void Init(IEOSCoroutineOwner coroutineOwner, string configFileName)
```

Comment:
I don't think this refactor should be done in this PR.

Reply:
The original implementation has been restored.